wgpu-core/src/device/mod.rs

```diff
@@ -3035,3 +3034,14 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
         }
     }
 }
+
+#[derive(Clone, Debug, Error)]
+pub enum CreateTextureError {
+    #[error("D24Plus textures cannot be copied")]
+    CannotCopyD24Plus,
+    #[error(
+        "texture descriptor mip level count ({0}) must be less than device max mip levels ({})",
+        MAX_MIP_LEVELS
+    )]
+    InvalidMipLevelCount(usize),
```

Comment:
we should use the same type we use for specifying the mipmap level count in the descriptor (`u32`?)